### PR TITLE
Add: Added Login and Signup Page

### PR DESCRIPTION
**Pull request by JaswanthReddyPadala**
Signup Page
image

Login Page
image

### CODE REVIEW

File: pages/login.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Login | QMart</title>
+    <link rel="stylesheet" href="https://quik-ui.netlify.app/style.css" />
+    <link rel="stylesheet" href="../qmart.css" />
+    <script
```
Comment (purohitdheeraj) [inline pages/login.html:10]: Here also remove this script tag

File: styles/login.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    width: 25%;
+    height: 100%;
+  }
+  
+  .q-login-input-container p {
+    color: var(--secondary-light);
+  }
+  
+  .q-login-input-container .input {
+    margin-top: 0.25rem;
```
Comment (purohitdheeraj) [inline styles/login.css:16]: 2.5px can be used instead of rem

File: styles/login.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    width: 100%;
+  }
+  
+  .q-login-options label,
+  .q-login-options label input {
+    margin: 0;
+    margin-right: 0.25rem;
```
Comment (purohitdheeraj) [inline styles/login.css:23]: here also px can be used

**Review by purohitdheeraj — APPROVED** [review]
Code looks Good
Good to merge ✌

File: styles/login.css [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  
+  .q-login-buttons .button-outline-primary {
+    border-color: var(--secondary-color);
+    width: 100%;
+    color: var(--secondary-color);
+  }
+
+
+  @media screen and (max-width: 988px) {
+    .q-login-main {
```
Comment (poornima-alugubelly) [inline styles/login.css:43]: Not sure if this was on purpose but u can use the standard media query check points like 992px , 748px as well

**Review by poornima-alugubelly — APPROVED** [review]

**Pull request merged**